plugins/explore/src/components/ExplorePage/ExploreTabs.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Tabs } from '@backstage/core';
+import { makeStyles } from '@material-ui/core';
+import React from 'react';
+import { DomainExplorerContent } from '../DomainExplorerContent';
+import { ToolExplorerContent } from '../ToolExplorerContent';
+
+// TODO: Support sub routes for these tabs in the future
```

Comment:
The `EntityPageLayout` found in the catalog uses the new pattern. We'll probably want to make a core variant of that that everybody can benefit from.

Reply:
I will extract it into `@backstage/core`

Reply:
Move it to @backstage/core and it works quite well. But I'm unable to display it in storybook. What kind of wrapper do I need around it? It can't discover the `Component.Route` as children.

Reply:
Found it, looks like `child.type !== Route` isn't always a safe operation, especially in storybook, but I found a replacement.